packages/evm/README.md

```diff
@@ -138,10 +138,10 @@ It is possible to individually activate EIP support in the EVM by instantiate th
 
 ```typescript
 import { Chain, Common } from '@ethereumjs/common'
-import { VM } from '@ethereumjs/vm'
+import { EVM } from '@ethereumjs/evm'
 
 const common = new Common({ chain: Chain.Mainnet, eips: [2537] })
-const vm = new VM({ common })
+const vm = new EVM({ common })
```

Comment:
Ah, no, this is not correct, read the initial sentence from this section. We cannot instantiate a standalone EVM right now (and this code just would not work), therefore the example with the VM.